app/assets/javascripts/components/menu_item.js.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var MenuItem = React.createClass({
+
+  navigate: function(hash) {
+    window.location.hash = hash;
+  },
+
+  render: function() {
+    var classStyle = "menu-item";
+    if (this.props.hash === "share-facebook" || this.props.hash === "get-started") {
+      classStyle += " first-item";
```

Comment:
I like your thinking here, but this problem has already been solved by [this](https://github.com/JedWatson/classnames) utility module. Try to use that module as much as possible.